draft_editor/draft_editor.talon

```diff
@@ -1,5 +1,5 @@
 user.draft_editor_running: True
-not app: draft_editor
+not tag: user.draft_editor_app_focused
```

Comment:
Since you're removing the mode, maybe you could remove the scope (user.draft_editor_running) as well? You could do this using:

```
tag: user.draft_editor_running
and not tag: user.draft_editor_app_focused
```

Reply:
What's wrong with the scope?

Reply:
The scope works fine, it's more my impression of the semantics. My feeling about scopes is that they're for passing more complex data to matchers. Since this is just a boolean it could be a tag. Kind of like how you replaced the app name (which wasn't referring to just one app and was dynamic) with a tag; it fits better with the Talon semantics. Do you agree with that impression?

Incidentally it looks like there are only two scopes in the repo currently (one is this one).

Reply:
I believe @AndreasArvidsson has argued for making both of them scopes. Do either of you feel strongly enough to push for one direction over the other?

Reply:
I don't feel strongly one way or the other.

Reply:
@splondike Ok now that this PR has been tested on all 3 platforms, I'm a bit hesitant to mess with it anymore. Are you ok with me just leaving a quick TODO comment and merging in? Maybe we can refactor some more next time we revisit, eg to combine it with the talon draft editor stuff?

Reply:
Yeah, I don't mind if you'd like to merge it in as is.